financial-connections/src/main/java/com/stripe/android/financialconnections/features/networkinglinkloginwarmup/NetworkingLinkLoginWarmupViewModel.kt

```diff
@@ -97,6 +97,8 @@ internal class NetworkingLinkLoginWarmupViewModel @AssistedInject constructor(
             lookupAccount(
                 pane = PANE,
                 email = payload.email,
+                phone = null,
+                phoneCountryCode = null,
```

Comment:
Would we need to pass the actual values here in case of attestation errors in the lookup call (if those can occur)?

Reply:
Likely no. Feel free to ignore.